src/components/overblikk.tsx

```diff
@@ -3,31 +3,80 @@ import {useState, useEffect } from "react";
 import "./overblikk.css"
 import hentVeileder from "../data/api/hentVeileder";
 import { VeilederData } from "../data/api/datatyper/veileder";
+import hentPersonalia from "../data/api/hentPersonalia";
+import { PersonaliaV2Info } from "../data/api/datatyper/personalia";
+import { OppfolgingsstatusData } from "../data/api/datatyper/oppfolgingsstatus";
+import { OrdinaerRegistrering, RegistreringsData } from "../data/api/datatyper/registreringsData";
+import hentOppfolgning from "../data/api/hentOppfolgning";
+import hentRegistrering from "../data/api/hentRegistrering";
+import { YtelseData } from "../data/api/datatyper/ytelse";
+import hentYtelser from "../data/api/hentYtelser";
 
 
 const Overblikk = () => {
 
-    const [veileder, setVeileder] = useState(0)
+    const [veileder, setVeileder] = useState<VeilederData | null>(null);
+    const [person, setPerson] = useState<PersonaliaV2Info | null>(null);
+    const [oppfolgning, setOppfolgning] = useState<OppfolgingsstatusData | null>(null);
+    const [registrering, setRegistrering] = useState<RegistreringsData | null>(null);
+    const [ytelser, setYtelser] = useState<YtelseData | null>(null);
 
 
     useEffect(() => {
-
       hentVeileder('Z123456').then(data => {
         setVeileder(data)
       });
     },[]);
-    
-
 
+    useEffect(() => {
+      hentPersonalia('10108000398').then(data => {
+        setPerson(data)
+      });
+    },[]);
 
+    useEffect(() => {
+      hentOppfolgning('10108000398').then(data => {
+        setOppfolgning(data)
+      });
+    },[]);
+    useEffect(() => {
+      hentRegistrering('10108000398').then(data => {
+        setRegistrering(data)
+      });
+    },[]);
+    useEffect(() => {
+      hentYtelser('10108000398').then(data => {
+        setYtelser(data)
+      });
+    },[]);
+    
 
   return (
     <Panel border  className="Panel">
-      <Heading spacing level="2" size="large">
+      <Heading spacing level="2" size="large" >
         Overblikk
       </Heading>
       <BodyLong>
-        <h3>Veileder: {veileder.navn} </h3>
+        <h3>Telefon: </h3>
+        <p>{person?.telefon?.[0]?.telefonNr}</p>
```

Comment:
Dersom vi tenker at komponenten (`Overblikk`) er en "dum" komponent (aka. en visningskomponent) så bør den ikke forholde seg til datahenting og domeneobjekter som backenden "eier." Ved å innføre props som er tilpasset akkurat den typen data som komponenten er interessert i så kan vi unngå dyp property-aksessering i domeneobjekter som kommer fra backenden, samt at vi kan løfte datahenting ut av komponenten og heller sende det ned via props. Da blir komponenten enklere for andre å resonnere rundt hva den gjør, samt at vi ikke trenger å endre komponenten dersom vi mot formodning skulle måtte endre må måten vi henter/mapper data på (da dette vil gjøres utenfor komponenten selv).